packages/blockchain/celo/src/lib/services/celo.web3.ts

```diff
@@ -0,0 +1,32 @@
+import { evmBasedWeb3 } from '@tatumio/shared-blockchain-evm-based'
+import Web3 from 'web3'
+import { EvmBasedBlockchain, httpHelper, THIRD_PARTY_API } from '@tatumio/shared-core'
+
+export const celoWeb3 = (args: { blockchain: EvmBasedBlockchain }) => {
+  const evmBasedWeb3Result = evmBasedWeb3(args)
+
+  return {
+    ...evmBasedWeb3Result,
+    getClient(provider?: string, fromPrivateKey?: string): Web3 {
+      const web3 = evmBasedWeb3Result.getClient(provider)
```

Comment:
Not sure but I can use Web3 lib for CELO as well - https://docs.celo.org/developer-guide/contractkit/notes-web3-with-contractkit .